orttraining/orttraining/core/session/training_session.h

```diff
@@ -135,14 +135,32 @@ class TrainingSession : public InferenceSession {
     // If not provided, no optimizer is added.
     optional<OptimizerConfiguration> optimizer_config{};
 
+    // struct to describe a specific edge (not node_arg). This information is
```

Comment:
>struct to describe a specific edge (not node_arg) [](start = 7, length = 49)

struct to describe a specific edge (not node_arg) ---> struct to describe edges associated to the same tensor. Note that one edge is a connection from a tensor to an operator. If a tensor is consumed by multiple operators, there would be multiple edges from that tensor to its consumers.